sdk/search/perf-tests/search-documents/test/getDocumentsCount.spec.ts

```diff
@@ -0,0 +1,251 @@
+              filterable: true,
+              sortable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.String",
+              name: "country",
+              searchable: true,
+              filterable: true,
+              sortable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.String",
+              name: "postalCode",
+              searchable: true,
+              filterable: true,
+              sortable: true,
+              facetable: true
+            }
+          ]
+        },
+        {
+          type: "Collection(Edm.ComplexType)",
+          name: "rooms",
+          fields: [
+            {
+              type: "Edm.String",
+              name: "description",
+              searchable: true,
+              analyzerName: KnownAnalyzerNames.EnLucene
+            },
+            {
+              type: "Edm.String",
+              name: "descriptionFr",
+              searchable: true,
+              analyzerName: KnownAnalyzerNames.FrLucene
+            },
+            {
+              type: "Edm.String",
+              name: "type",
+              searchable: true,
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.Double",
+              name: "baseRate",
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.String",
+              name: "bedOptions",
+              searchable: true,
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.Int32",
+              name: "sleepsCount",
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Edm.Boolean",
+              name: "smokingAllowed",
+              filterable: true,
+              facetable: true
+            },
+            {
+              type: "Collection(Edm.String)",
+              name: "tags",
+              searchable: true,
+              filterable: true,
+              facetable: true
+            }
+          ]
+        }
+      ],
+      suggesters: [
+        {
+          name: "sg",
+          searchMode: "analyzingInfixMatching",
+          sourceFields: ["description", "hotelName"]
+        }
+      ],
+      scoringProfiles: [
+        {
+          name: "nearest",
+          functionAggregation: "sum",
+          functions: [
+            {
+              type: "distance",
+              fieldName: "location",
+              boost: 2,
+              parameters: {
+                referencePointParameter: "myloc",
+                boostingDistance: 100
+              }
+            }
+          ]
+        }
+      ],
+      corsOptions: {
+        // for browser tests
+        allowedOrigins: ["*"]
+      }
+    };
+    await this.searchIndexClient.createIndex(hotelIndex);
```

Comment:
So, here we are creating an index.
And calling getDocumentsCount in the run method.

I'm assuming that an index is a document. Am I wrong?
What do we expect the getDocumentsCount to return in this case?